src/index.js

```diff
@@ -554,7 +554,17 @@ class EliteDangerousJournalServer {
     if (event === 'change' && filepath === this.currentJournal) {
       this.getJournalUpdate();
     } else {
-      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occured for`)} ${chalk.magenta(filepath)}`);
+      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occurred for`)} ${chalk.magenta(filepath)}`);
+      const baseName = path.basename(filepath).toLowerCase();
+      if (baseName === 'status.json') {
```

Comment:
Should we pop the `'status.json'` String into a constant at the top of the file to make it easier to adjust to any changes in the future? Or should we go ahead and create an Array in case they add any other files like this and just check if the Array includes the `baseName` before broadcasting?

Reply:
Or rather, pop it into the `src/constants/index.js` file.

Reply:
The array method is more suitable. Allows also to specify the event name to be used to send such data. Here's the list of files that are rewritten:

* Cargo
* Market
* ModulesInfo
* Outfitting
* Shipyard
* Status

Most of these (Cargo, Market, ModulesInfo, Outfitting, Shipyard) write a small entry in the journal and write the full entry in a separate file. I would suggest the Websocket broadcasts the small entry as `{ "event": "Cargo", ... }` and the file contents with all the info maybe as `{ "event":"X-Cargo", ..., ..., ... }`. Status could be sent as `"X-Status"` with this scheme too, for consistency. What do you think?

Reply:
Ah, I didn't realize there were already a bunch of other files that operate this way.

Just wondering, is there any significance to the `X-*` prefix?

Reply:
When ED emits the "Cargo" event, it appends a "reduced" event to the journal, and writes the "complete" event into the ancillary file. To distinguish between the two (and support subscribing to one, but not the other), the events should have a difference. Prefixing with "X-" seemed like something that's typically done in the web world, for example when adding custom headers to emails or http requests.

How about this: It's the e**X**tended event, since it contains all fields.

Reply:
👍 Works for me. Just wasn't sure if there was something specific to the Journal updates that I wasn't aware of.